.github/workflows/ci.yaml

```diff
@@ -0,0 +1,42 @@
+name: 'Continuous Integration'
+
+on:
+  workflow_call:
+    secrets:
+      AWS_ACCESS_KEY_ID:
+        required: true
+      AWS_SECRET_ACCESS_KEY:
+        required: true
+
+permissions:
+  contents: read
+
+jobs:
+  integration-tests:
+    runs-on: ubuntu-latest
+    env:
+      AWS_ACCESS: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+      ENVIRONMENT: qa
+    steps:
+      - uses: actions/checkout@v2
```

Comment:
We should use checkout@v4 since v2 is running on a deprecated version of node.

Reply:
good catch - upgrading!